core/src/main/scala/hydra/core/monitor/HydraMetrics.scala

```diff
@@ -70,7 +74,9 @@ object HydraMetrics {
     getOrCreateHistogram(lookupKey, metricName, tags).map(_.record(value))
   }
 
-  def addPromHttpMetric(topic: String, responseCode: String, path: String)(implicit ec: ExecutionContext): Unit = {
+  def addHttpMetric(topic: String, responseCode: String, path: String,
```

Comment:
What about having `responseCode` here be of type `StatusCode` rather than doing `.toString` on the status code every time we use this?

Reply:
It is something we should have done when this was first created. I agree and will change.